netdata.spec.in

```diff
@@ -342,8 +342,13 @@ install -m 750 -p packaging/installer/netdata-uninstaller.sh \
 
 # ###########################################################
 # Install netdata service
+
 install -m 755 -d "${RPM_BUILD_ROOT}%{_unitdir}"
+%if 0%{?centos_ver} != 7 && 0%{?amzn} != 2
```

Comment:
Should it be `amazon_linux`? I CTRL-F and found only `0%{?amazon_linux}` in this file.

Reply:
Quite so, I forgot that they changed their macro names between versions.